logical-comaprison-operators.js

```diff
@@ -6,36 +6,38 @@ TASK 1:
  // Have fun! // 😃
 ********************************************************************************/
 
-const exp1 = 10 >= 10; // TODO: ADD YOUR EVALUATION HERE -->
```

Comment:
You were supposed to write the evaluation outcome inside a comment to test your knowledge, please do when you can.